docs/knowledge-base/release-management/release-management.mdx

```diff
@@ -0,0 +1,208 @@
+Using the new strategy, we make sure that there will be one source of truth and it sits in helm-charts outside the internal network and only there! 
+
+4. **Simpler deployment**  
+IF we keep them together, we will need to manage every chart (along with the values) in different branches. 
+That way we will have hard time to manage the charts in all the branches.
+With the new strategy, we need to "worry" only the values and the version of the chart when we deploy. We won't manage the chart anymore.  
+We can only run this command template:  
+`helm install myrelease oci://HELM_REGISTRY/helm-charts/raster-ingestion --version 3.1.0`  
+And that's it! We don't need the chart in the repo anymore. We make the structure and the config independent to each other.
+
+Let's walk through the flows!
+
+### Chart Flow
+
+![Chart Flow](./images/chart-flow.png)
+
+This flow suppose to handle the update of new versions of services and creation of new "Libot Release".
+
+1. When a service has a new release and helm folder (points that its chart in helm-charts needs to be updated), 
+open a PR in helm-charts that update the version of the service.
+
+:::note
+The developer should see the PR and make more changes if necessary
+:::
+
+2. After the PR is ready, merge the PR.
+
+::::info
+The PR can contain multiple commits so if there are more than one service that triggered the PR (more than one chart that needs to be updated), 
+the PR will contain all the changes so the merge of the PR will update the versions of all the relevant services in helm-charts.
+
+:::tip
+If there is a service that you don't want to include it yet in helm-charts, you can disable the workflow that triggers that PR
+:::
+
+::::
+
+3. On "master" push:
+    - Create a new "Libot Version" using release-please. 
+    The release-please should also update the docker and helm versions of the services in:
+        - `artifacts.json` file inside the domain that represent the information of all services and versions of the domain
+        - `libot-artifacts.json` in the root directory that represent the information of all services and versions of **all** domains
+
+    - Package all the charts and push them to helm registry.  
+    For example:  
+    On new libot release with version 3.0.1, we will push to helm registry the raster-ingestion chart with this name: `rater-ingestion-3.0.1`. 
+
+**End State**:
+
+* New "Libot Version" 
+* Charts of the new version packed in helm registry.
+
+Now, we have an updated structure of the chart and we can move on to the values.
+
+### Values Flow
+
+![Values Flow](./images/values-flow.png)
+
+This flow suppose to handle the update of the namespaces values (integration and prod) and the deployment in openshift.
+
+0. On new release, there will be a PR on stage branch in `helm-values` repo that updates the "Libot Version" in `libot-version.json` file
```

Comment:
Start from `1`.

Reply:
This is 1 on purpose because this is a stage coming from the previous flow